packages/base-styles/_z-index.scss

```diff
@@ -32,7 +32,6 @@ $z-layers: (
 	".edit-post-layout__footer": 30,
 	".interface-interface-skeleton__header": 30,
 	".interface-interface-skeleton__content": 20,
-	".edit-site-header": 62,
```

Comment:
It seems these z-index layers were not being used even before this PR. It would be nice to have something that detects this.